tests/data_files/Makefile

```diff
@@ -318,6 +318,15 @@ rsa_pkcs8_2048_public.der: rsa_pkcs8_2048_public.pem
 	$(OPENSSL) rsa -pubin -in $< -outform DER -pubout -out $@
 all_final += rsa_pkcs8_2048_public.der
 
+authorityKeyId_subjectKeyId.crt:
+	$(OPENSSL) req -x509 -nodes -days 7300 -newkey rsa:2048 -keyout authorityKeyId_subjectKeyId.crt -out authorityKeyId_subjectKeyId.crt -config authorityKeyId_subjectKeyId.conf -extensions 'v3_req'
+# The listed certificates are the copies of authorityKeyId_subjectKeyId.crt with error injections
+# authorityKeyId_subjectKeyId_wrong_SubjectKeyId.crt              The TAG marking the beginning of SubjectKeyId is set to 0x00
```

Comment:
It would be great to have makefile rules to create each of these files. 
See https://github.com/Mbed-TLS/mbedtls/pull/6900/files#diff-33f05cba6c4be0b2bf38b859d772b1a0527c9d80ad7347aada68eee172bb584aR297 for reference.

Reply:
Yes, I agree that this would be better solution, but this would require generating new certs and malformed versions.
I suggest to leave it for now and create issue for this.